service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/DocumentReferenceToNarrativeStatementMapper.java

```diff
@@ -63,6 +63,16 @@ public String mapDocumentReferenceToNarrativeStatement(final DocumentReference d
         return TemplateUtils.fillTemplate(NARRATIVE_STATEMENT_TEMPLATE, builder.build());
     }
 
+    public String buildBindingDocumentNarrativeStatement(String bindingDocumentId) {
```

Comment:
The "binding document" was my invention. Let's use the "fragment index" that is used in the examples